ChronologBackend/nlp.py

```diff
@@ -2,45 +2,58 @@
 import spacy
 import json
 
-# TODO: Determine and justify this value (currently a dummy value)
-NUM_SPLITS = 10
+JSON_DIRECTORY = "timelines/"
 
 
-def process_data(text, chapter_regex):
+def process_data(text, chapter_regex, num_splits, quiet):
     """
     :param text: The full text to be analysed.
     :param chapter_regex: Regex by which chapters are determined. If chapter_regex=="", split into equal length sections.
+    :param quiet: Flag to silence print statements
     :return: Returns list of sections
     """
+    if not num_splits:
```

Comment:
In the chronolog main, we pass in `args.sections or DEFAULT_SPLITS` into the`num_splits` parameter so it should never be `None`. So I think we should remove this check.